common/src/main/java/com/wynntils/models/emeralds/EmeraldModel.java

```diff
@@ -191,7 +201,13 @@ public void onContainerSetContent(ContainerSetContentEvent.Pre event) {
         }
 
         for (int i = 0; i < containerStop; i++) {
-            adjustBalance(items.get(i), 1, false);
+            ItemStack is = items.get(i);
+            adjustBalance(is, 1, false);
+            // We need to make a map of the items in the container with emerald values for when the duplicate SetSlot
+            // event is received
```

Comment:
Can you manually split this long comment into 2 lines that are not this weird to read?